srcflib/plumbing/bespoke.py

```diff
@@ -50,6 +50,25 @@ def context(sess: SQLASession = None) -> Generator[SQLASession, None, None]:
         sess.commit()
 
 
+class MailHandler(Enum):
+    """
+    Choices for handling of email sent to `@srcf.net` addresses.
+    """
+
+    forward = 1
+    """
+    Forward emails to the user's registered contact address.
+    """
+    pip = 2
+    """
+    Process emails using Exim.
+    """
+    hades = 3
+    """
+    Deliver emails to the user's Hades mailbox.
+    """
```

Comment:
Can we have a good way of keeping these values in strict sync with `srcf.database.schema`?

Reply:
Yeah, this should probably replace `srcf.database.schema.VALID_MAIL_HANDLERS`:

```python
class Member:
    ...
    mail_handler = Column(Enum(*(handler.name for handler in MailHandler)))
```

(`Enum(MailHandler)` requires SQLAlchemy 1.1 -- Soon™)

Reply:
This has now moved to `srcf.database`, and is shared with `Member`.

We recently gained a newer SQLAlchemy, but I've skipped making it `Enum(MailHandler)` as we'll need changes elsewhere (control panel, and anywhere else that interacts with this field) to handle instances of `MailHandler` rather than plain `str` -- best done separately.